README.md

```diff
@@ -1,61 +1,246 @@
 This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
 
-## Table of contents
-TBA
-
-## Team Atlantic Developer guide
-
-### Dependcies + development
-Install Dependencies:
+# Team Atlantic Developer Guide
```

Comment:
Can change this to the title of the application

Reply:
updated